src/Cardano/AddressV3.h

```diff
@@ -91,8 +91,3 @@ inline bool operator==(const AddressV3& lhs, const AddressV3& rhs) {
 }
 
 } // namespace TW::Cardano
-
-/// Wrapper for C interface.
-struct TWCardanoAddress {
```

Comment:
Is this change intended to be here?

Reply:
This interface is not used at all, I forget to delete it in previous pr